ADR/0044-spdx-support.md

```diff
@@ -0,0 +1,162 @@
+
+### CycloneDX -> SPDX conversion
+
+CycloneDX (1.4) is structured document in json format with following structure (not full specification)
+
+- Document Root
+    - Metadata
+        - Tools
+            - `List<Tool>`
+                - vendor
+                - name
+    - Components
+        - `List<Component>`
+            - name
+            - version
+            - purl
+            - properties
+            - `List<Property>`
+                - name
+                - value
+    - formulations
+        - `List<Component>`
+
+SPDX (2.3) is structured document in json format with following structure(not full specification):
+- Document Root
+    - name
+    - SPDXID
+    - creationInfo
+        - Creators
+            - `List<String>`
+    - packages
+        - `List<Packages>`
+            - SPDXID
+            - name
+            - versionInfo
+            - externalRefs
+                - `List<ExternalRef>`
+                    - referenceCategory
+                    - referenceType
+                    - referenceLocator
+            - annotations
+                - `List<Annotation>`
+                    - annotationDate
+                    - annotationType
+                    - annotator
+                    - Comment
+            - relationships
+                - `List<Relationship>`
+                    - spdxElementId
+                    - relationshipType
+                    - relatedSpdxElement
+
+#### 1:1 conversions
+Following CycloneDX to SPDX attributes are converted as 1:1 as they represent the same thing.
+
+| CycloneDX Attribute        | SPDX Attribute      |
+|----------------------------|---------------------|
+| components                 | packages            |
+| component.name             | package.name        |
+| component.version          | package.versionInfo |
+
+
+#### Component.purl
+CycloneDX (version 1.4) supports only a single purl attribute per component. SPDX doesn’t have a direct attribute, but instead every package includes an externalRefs array which describes all external references for the package. There are defined reference categories and types. For PURL, category PACKAGE-MANAGER and type purl is used. The purl itself will be stored as referenceLocator
+
+| CycloneDX Attribute          | SPDX Attribute                                                |
+|------------------------------|---------------------------------------------------------------|
+| component.purl = `<PURL>`    | package.externalRefs = [{referenceCategory:”PACKAGE-MANAGER”, |
+|                              |                          referenceType:purl,                  |
+|                              |                          referenceLocator: `<PURL>`           |
+|                              |                          }]                                   |
```

Comment:
I believe this was fixed. All tables look correct to me

Reply:
The loss of indentation hurts readability, especially for this one

![image](https://github.com/user-attachments/assets/4e2dc48e-b4f7-4e65-ac0b-9b6c56a09b87)

How about changing them from Markdown tables to monospace blocks? Or just wrapping the tables in `` ``` `` so that they look the same as in the source code